oemoflex/postprocessing.py

```diff
@@ -565,17 +969,25 @@ def run_postprocessing(year, name, exp_paths):
     # costs
     varom_cost = get_varom_cost(oemoflex_scalars, prep_elements)
     carrier_cost = get_carrier_cost(oemoflex_scalars, prep_elements)
-    fuel_cost = get_fuel_cost(carrier_cost, scalars_raw)
-    emission_cost = get_emission_cost(carrier_cost, scalars_raw)
+    fuel_cost = get_fuel_cost(carrier_cost, prep_elements, scalars_raw)
+    emission_cost = get_emission_cost(carrier_cost, prep_elements, scalars_raw)
     aggregated_emission_cost = aggregate_by_country(emission_cost)
+    invest_cost = get_invest_cost(oemoflex_scalars, prep_elements, scalars_raw)
+    fixom_cost = get_fixom_cost(oemoflex_scalars, prep_elements, scalars_raw)
     oemoflex_scalars = pd.concat([
-        oemoflex_scalars, varom_cost, carrier_cost, fuel_cost, aggregated_emission_cost
+        oemoflex_scalars, varom_cost, carrier_cost, fuel_cost, aggregated_emission_cost,
+        emission_cost,
```

Comment:
When concatenating emission_cost and aggregated_emission_cost in oemoflex_scalars, the costs appear twice. Before, emission costs where reported only in aggregated form. If there is no strong reason for doing it different, I suggest to remove this line.

Reply:
I removed the line in https://github.com/modex-flexmex/oemo-flex/pull/48/commits/a09ec8648354566c1d9b1c0879aa334b285d6e90